dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/NettyRemoteChannel.java

```diff
@@ -43,13 +45,21 @@
      */
     private final Host host;
 
+    private final Random random = new Random();
+
 
     public NettyRemoteChannel(Channel channel, long opaque) {
         this.channel = channel;
         this.host = ChannelUtils.toAddress(channel);
         this.opaque = opaque;
     }
 
+    public NettyRemoteChannel(Channel channel) {
+        this.channel = channel;
+        this.host = ChannelUtils.toAddress(channel);
+        this.opaque = random.nextLong();
```

Comment:
opaque should be -1 is much better.  we can locate the issue easily.

Reply:
ok